.changelog/9978.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:bug
-cache: fix a bug in the client agent cache where streaming could potentially leak resources. [[GH-9978](https://github.com/hashicorp/consul/pulls/9978)].
+cache: fix a bug in the client agent cache where streaming could potentially leak resources. [[GH-9978](https://github.com/hashicorp/consul/pull/9978)].
```

Comment:
I fixed this in the upstream PR branch too and reset base but it's still showing here 🤷